test/k8sT/microscope.go

```diff
@@ -39,6 +39,8 @@ var _ = Describe("K8sMicroscope", func() {
 	})
 
 	AfterAll(func() {
+		kubectl.CiliumReport(helpers.KubeSystemNamespace,
```

Comment:
I know that this is wip, but if for each AfterAll we execute this, a few GB will be storage per each build. Should be in the afterFailed. ;-)